test/e2e-v2/cases/mariadb/mariadb-slowsql/docker-compose.yaml

```diff
@@ -0,0 +1,124 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  oap:
+    extends:
+      file: ../../../script/docker-compose/base-compose.yml
+      service: oap
+    image: ghcr.io/apache/skywalking/oap:bcd9f7a56b99ca612711ed3a540bdae5f46e9171
```

Comment:
```suggestion
```
The oap image version should be controled in `base-compose`

Reply:
OAP should be latest build, rather than an old commit ID.
@yswdqz Does your test have similar case?

Reply:
> OAP should be latest build, rather than an old commit ID.
> @yswdqz Does your test have similar case?

Yes, I think it should be latest build. `Image` should be removed.

Reply:
TTL cases have all down. @yswdqz I don't know why. Could you recheck on the master branch as well?

Reply:
I get it.

Reply:
I think I have stabled the TTL tests. They don't block this PR for now.